packages/conf-gmp-powm-sec/conf-gmp-powm-sec.3/opam

```diff
@@ -4,7 +4,8 @@ homepage: "http://gmplib.org/"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "GPL-1.0-or-later"
 build: [
-  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos"}
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos" & os != "win32" }
+  ["sh" "-exc" "$(ocamlc -config-var c_compiler) -c $CFLAGS test.c"] {os = "win32"}
```

Comment:
opam-repository-mingw's depext model required the compiler to have been already installed (which worked, as it was using the opam-depext plugin). I'm afraid while it's a noble goal to want to use the same C compiler as ocamlc/ocamlopt, it's not guaranteed to be there.
```suggestion
  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos" & os != "win32"}
  ["sh" "-exc"
    "%{host-arch-x86_64:installed?x86_64:}%%{host-arch-x86_32:installed?i686:}%-w64-mingw32-gcc -c $CFLAGS test.c"
  ] {os = "win32"}
```